### PR TITLE
Added docs.

### PR DESCRIPTION
**Pull request by ashryanbeats**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,20 +24,85 @@ phonegap-plugin-csdk-client-auth
...
+
+Client Auth is required for all Creative SDK integrations. See the information below for info on using this Client Auth plugin.
+
+### Contents
+
+- [Prerequisites](#prereqs)
+- [Installation](#install)
+- [Setup guide](#setup)
+- [API guide](#api)
+
```
Comment (macdonst) [inline README.md:37]: You can change the above to:

```
[Prerequisites](#prerequisites)
[Installation](#installation)
[Setup guide](#setup-guide)
[API guide](#api-guide)
```

File: README.md [hunk 2]
```diff
@@ -24,20 +24,85 @@ phonegap-plugin-csdk-client-auth
...
+
+<a name="prereqs"></a>
```
Comment (macdonst) [inline README.md]: Can be deleted.

File: README.md [hunk 3]
```diff
@@ -24,20 +24,85 @@ phonegap-plugin-csdk-client-auth
...
+
+**Android**
+
+- Android Studio 2.0 or higher
+- Android SDK 16 or higher
+
+[See the Creative SDK Getting Started guide for further details](https://creativesdk.adobe.com/docs/android/#/articles/gettingstarted/index.html#prerequisites).
+
+
+<a name="install"></a>
```
Comment (macdonst) [inline README.md]: Can be deleted.

File: README.md [hunk 4]
```diff
@@ -24,20 +24,85 @@ phonegap-plugin-csdk-client-auth
...
+
+The ZIP files contain all the frameworks in the Creative SDK, but for this plugin we will only be using the `AdobeCreativeSDKCore.framework`.
+
+
+**Android** 
+
+No action is required for Android. The Creative SDK for Android is delivered as a remote Maven repository, and the required framework will be downloaded automatically by the plugin.
+
+
+<a name="setup"></a>
```
Comment (macdonst) [inline README.md]: Can be deleted.

File: README.md [hunk 5]
```diff
@@ -24,20 +24,85 @@ phonegap-plugin-csdk-client-auth
...
+# Setup guide
+
+1. Create a new PhoneGap app or `cd` into an existing one
+1. Add this plugin (see "Adding the plugin" above)
+1. **iOS only:** download and add the Creative SDK to this plugin's `src/ios` directory (see "Downloading the Creative SDK" above)
+1. Add your target PhoneGap platform
+1. Build and run for your platform
+
+
+<a name="api"></a>
```
Comment (macdonst) [inline README.md:104]: Can be deleted and add:

```
# API Guide
```

**Comment by ashryanbeats** [comment]
@macdonst Updated anchor links for gfm!

**Pull request merged**